plugin/core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -42,6 +42,12 @@ class BintrayConfiguration {
                     gpg {
                         sign = extension.sign
                     }
+
+                    mavenCentralSync {
+                        sync = extension.mavenCentralSync
```

Comment:
maybe we can simplify this and assume that
```
sync = propertyFinder.mavenCentralUser && propertyFinder.mavenCentralPassword
```

Reply:
@mr-archano that would work, but I see the use-case where you have the env. set but you don't wanna sync., maybe because of release/snapshots deployments, what do you think? the current impl. is verbose, but it works for all the cases.

Reply:
good point, let's keep it like this then - after all it's closer to the original implementation too

Reply:
thanks :)